core/src/sumeragi/main_loop.rs

```diff
@@ -535,14 +532,14 @@ fn enqueue_transaction(sumeragi: &Sumeragi, wsv: &WorldStateView, tx: VersionedS
 }
 
 #[allow(clippy::too_many_lines)]
-fn handle_message(
+fn handle_message<const IS_GENESIS: bool>(
```

Comment:
I think this function shouldn't have const generic as it's called only in non-genesis rounds.

Reply:
what about [this](https://github.com/hyperledger/iroha/pull/3356#discussion_r1160039422)

Reply:
Sounds reasonable to me.